agent/bench-scripts/pbench-uperf

```diff
@@ -743,15 +740,13 @@ done
 echo "$script_path/postprocess/generate-benchmark-summary \"$benchmark\" \"$orig_cmd\" \"$benchmark_run_dir\"" >"$benchmark_run_dir/generate-benchmark-summary.cmd"
 chmod +x "$benchmark_run_dir/generate-benchmark-summary.cmd"
 $benchmark_run_dir/generate-benchmark-summary.cmd
-pbench-end-tools --group=$tool_group --dir=$benchmark_run_dir
-pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir end
 
-pbench-collect-sysinfo --group=$tool_group --dir=$benchmark_run_dir --sysinfo=$sysinfo end
-
-# Stop the tool meisters on each registered local/remote host
-pbench-tool-meister-stop
-if [[ ${?} != 0 ]]; then
-	error_log "[${script_name}]: failed to stop the tool meisters."
+if [[ $postprocess_only == "n" ]]; then
```

Comment:
This is a bug fix, because if post-process only was requested we still tried to run the Tool Meisters.